librz/analysis/serialize_analysis.c

```diff
@@ -13,13 +13,13 @@
  *
  * /
  *   /blocks
- *     0x<addr>={size:<ut64>, jump?:<ut64>, fail?:<ut64>, traced?:true, folded?:true, colorize?:<ut32>,
```

Comment:
@thestr4ng3r I think we can consider the project format changeable until 1.0.0, right?